### ISSUE TITLE
Does not work in multiplayer

### ISSUE THREAD

**Issue by RRRu10N** [AstemirDev/BedrockCameraCommand#2]
Free camera by key is not working. Camera modes is not showing. 
When trying to execute the command on a player he gets kicked from server. 
Mod starts to work after you go in single player execute the command and join back to server. But its starts to work only for players who did this, others will still crash.
This is not modpack`s reason
Is there any way to fix that? cuz mod is very good
![image](https://github.com/AstemirDev/BedrockCameraCommand/assets/169887358/19f78bf8-53ef-4242-bcc8-d41231ac5d94)
![image](https://github.com/AstemirDev/BedrockCameraCommand/assets/169887358/fadc3b87-c340-491d-853b-381a56fbd9dd)

**Comment by Jetpack-Cat** [comment]
Well that sucks. Hope it can get fixed because I'm building several modpacks with this mod, I hope I don't have to remove it because it's amazing

**Comment by power-er** [comment]
Have you found how to fix the problem without exiting?

> Free camera by key is not working. Camera modes is not showing. When trying to execute the command on a player he gets kicked from server. Mod starts to work after you go in single player execute the command and join back to server. But its starts to work only for players who did this, others will still crash. This is not modpack`s reason Is there any way to fix that? cuz mod is very good ![image](https://private-user-images.githubusercontent.com/169887358/330924281-19f78bf8-53ef-4242-bcc8-d41231ac5d94.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4NDg5MjQsIm5iZiI6MTcyNDg0ODYyNCwicGF0aCI6Ii8xNjk4ODczNTgvMzMwOTI0MjgxLTE5Zjc4YmY4LTUzZWYtNDI0Mi1iY2M4LWQ0MTIzMWFjNWQ5NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyOFQxMjM3MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNjY0YzIxYzU2YmI5YTEwYWFmOGQ0YzVjMjkxMGM5NDY3ZTFiZmNhNTNlYmQyMDMyNmJjM2ZkOTVhM2E4ZGE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.v3kkzDyhO8G-nPaB2WjsZEvEnFFfGKFSlLKAPfrLim0) ![image](https://private-user-images.githubusercontent.com/169887358/330924701-fadc3b87-c340-491d-853b-381a56fbd9dd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4NDg5MjQsIm5iZiI6MTcyNDg0ODYyNCwicGF0aCI6Ii8xNjk4ODczNTgvMzMwOTI0NzAxLWZhZGMzYjg3LWMzNDAtNDkxZC04NTNiLTM4MWE1NmZiZDlkZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyOFQxMjM3MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YjMxZDM4NWRiYTQzNzQwOWI3OWI0NzEyMDZhMGU5MDI2ZmExNTY5YTVlOWZjMGNiODI1MTQ3ODZmZGNlYzFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.8MIFDT8P0evSKljeDaFCEA82o0R7w5PeYaJfJMNp5Q0)

**Comment by Miniontoby** [comment]
Yeah I also have the same problem. I think it is cause the camera presets aren't correctly being detected or something.

If you don't go to singleplayer first, then the command completions are missing the available presets on multiplayer, but after ONLY ENTERING singleplayer world and then going to multiplayer it does show the presets.

And given the error message that we get "Invalid camera mode: ...", is why I can make the assumption that the presets aren't loaded in correctly when you startup minecraft...

I hope to be able to fix this... but I cannot promise


Problem may be that the ServerCameraModeManager isn't loaded in...
